### DISCUSSION TITLE
Does this train the llm

### DISCUSSION THREAD

**Discussion by whiteice217** [nicobrauchtgit/obsidian-Smart2Brain#133] [Q&A]
I've been diving into AI recently and working on a project to help future nursing students. I run an open web UI server and created a model specifically for students with my own hosted Ollama docker. I've been trying to find a way to add my notes to it from class easily and was hoping I could use second brain in combination with my notes, and self-hosted system to do this. Currently, I have second brain using my system, and when I index notes does it store them on the Ollama system or are they just saved on my local device and the LLM pulls the vector data from my local device?

**Comment by Leo310** [comment]
I know Its been some time, but to answer your question, they are stored on your local device. 
Btw soon we will Release V2 of the plugin with a lot of enhancements ;)
Wont take long anymore!
